src/client/pages/Debugger/components/Offer.tsx

```diff
@@ -124,371 +197,93 @@ export const Offer: React.FC = () => {
-
-              {quoteType === QuoteType.DanishHome && (
-                <Formik
-                  initialValues={{
-                    firstName: 'Blargh',
-                    lastName: 'Blarghson',
-                    currentInsurer: '',
-                    birthDate: '1988-09-16',
-                    ssn: '',
-                    startDate: '',
-                    email: 'blargis@hedvig.com',
-                    danishHomeContents: {
-                      coInsured: 0,
-                      livingSpace: 34,
-                      street: 'Nørrebrogade 50',
-                      zipCode: '',
-                    },
-                  }}
-                  onSubmit={async (values) => {
-                    await createQuote(
-                      storage,
-                      localeIsoCode,
-                    )({
-                      input: {
-                        ...values,
-                        id: quoteId,
-                        currentInsurer: values.currentInsurer || undefined,
-                        // @ts-ignore
-                        startDate: values.startDate || undefined,
-                      },
-                    })
-
-                    if (refetch) {
-                      await refetch()
-                    }
-                  }}
-                >
-                  {(props) => (
-                    <>
-                      <QuoteForm formik={props}>
-                        <DanishHome formik={props} />
-                      </QuoteForm>
-                    </>
-                  )}
-                </Formik>
-              )}
-
-              {quoteType === QuoteType.NorwegianTravel && (
-                <Formik
-                  initialValues={{
-                    firstName: 'Blargh',
-                    lastName: 'Blarghson',
-                    currentInsurer: '',
-                    birthDate: '1995-09-29',
-                    ssn: '',
-                    startDate: '',
-                    email: 'blargis@hedvig.com',
-                    norwegianTravel: {
-                      coInsured: 0,
-                      isYouth: false,
-                    },
-                  }}
-                  onSubmit={async (values) => {
-                    await createQuote(
-                      storage,
-                      localeIsoCode,
-                    )({
-                      input: {
-                        ...values,
-                        id: quoteId,
-                        currentInsurer: values.currentInsurer || undefined,
-                        // @ts-ignore
-                        startDate: values.startDate || undefined,
-                      },
-                    })
+              <Formik
+                initialValues={
+                  getCurrentAvailableQuoteData(currentMarket, quoteType)
+                    ?.initialFormValues || {}
+                }
+                onSubmit={async (values) => handleSubmit(values, storage)}
```

Comment:
since we don't have any `await` this does not have to be async.

Reply:
Ah, I believe I thought that since `handelSubmit` is an `async` function we needed the `async` here as well. 😁 But you are of course very right, that's not how it works. I will remove that!